Data/pubtator_to_xml.py

```diff
@@ -0,0 +1,152 @@
+import gzip
+
+import tqdm
+
+
+def generate_annotation(file_object, tagged_term, index):
```

Comment:
Are you talking about lxml module? If so, I did use that before this pull request; however, it didn't scale and ended up crashing my computer.